bokeh/server/server.py

```diff
@@ -501,6 +513,10 @@ class _ServerOpts(Options):
     The port number the server should listen on for HTTP requests.
     """)  # type: ignore[assignment]
 
+    unix_socket : str | None = Nullable(String, help="""
+    The unix socket the server should bind to.
```

Comment:
Let's add a sentence here to the effect that when a unix socket is specified, the network/port options cannot be used.